Changes for variant products, links in DQI tab and new languages (#523)
akeneo/pim-helpcenter: content/md/what-is/understand-data-quality.md

- Lowercases formatting: the PIM checks whether text area attributes are correctly formatted for lower cases.

::: info

> Spell check is available for English, French, German, Spanish, Italian, Dutch, Danish, Swedish locales.

Reviewer: ```suggestion
Spell check is available for the English, French, German, Spanish, Italian, Dutch, Danish, and Swedish locales.
```

Spell check is usable with text and simple text area attributes.

> Spell check is usable with text, simple text area and rich text area attributes.

Reviewer: ```suggestion
Spell check works on text, simple text area and rich text area attributes.
```
